MIP/mip-40/README.md

```diff
@@ -0,0 +1,83 @@
+
+1. The L1 staking token MUST be a unique token issued in a fixed supply.
+2. The L2 gas token MUST be a unique token issued in a fixed supply.
+3. Rewards issued via FFS MUST NOT be issued in the L1 staking token, but instead in an LP token on the L1.
+4. The LP Token MAY elect to maintain a supply or value correlated with the L1 staking token or the L2 gas token. But, this supply or value correlation MUST NOT rely on special capabilities of the LP token or trusted third parties.
+5. The L2 staking token MUST be burned on the L2 to deposit gas into an L2 pool.
+
+If the LP Token supply is not fixed, the AB-FFS solution implemented is referred to as *AB-FFS Decoupled* Demurrage. [MIP-n](todo) provides a more complete specification for *AB-FFS Decoupled* Demurrage. 
+
+Otherwise, the AB-FFS solution implemented implemented is referred to as *AB-FFS Decoupled* Grant. [MIP-n](todo) provides a more complete specification for *AB-FFS Decoupled* Grant.
+
+### L1 LP Token
+1. The L1 LP Token MUST NOT enshrine any special capabilities to mint or otherwise unlock the L1 staking token. That is, the L1 LP token must be independent of the L1 staking token and only effectively be used to purchase the L1 staking token.
+2. The governing system MAY provide vouchers or other means of redeeming the L1 LP token for the L1 staking token from a dedicate allocation. The object of these vouchers SHOULD be to peg the value of the L1 LP token to the L1 staking token.
+
+A more complete proposal for the usage L1 LP tokens with FFS is provided in [MIP-n](todo).
+
+### L2 Token Pool (Gas Pool)
+1. Burning gas an amount of gas $\alpha$ on the L2 MUST result in depositing that same amount of gas $\alpha$ into an L2 pool.
+2. The L2 pool MUST be recirculated to the L2 users in a manner that is not specified in this MIP. Generally, this means the L2 pool must either be used to issue rewards or otherwise be a reliable means of purchasing gas token on the L2. 
+
+A system adhering to these requirements is referred to as a Gas Pool. A more complete proposal for the usage of Gas Pools is provided in [MIP-n](todo).
+
+### Intermediary Tokens
+1. A complete implementation of *AB-FFS Decoupled* SHOULD use intermediary tokens to bridge the L1 and L2 systems. An implementation not using intermediary tokens is referred to as *AB-FFS Partially Decoupled*.
+2. The intermediary tokens MAY manage their own exchange rates, markets, and fees between each other, the L1 staking token, the L2 staking token, and the L1 LP token.
+
+A more complete proposal for the usage of intermediary tokens is provided in [MIP-n](todo).
+
+### Genesis
+[MD-38](https://github.com/movementlabsxyz/MIP/pull/38) requests provisions for genesis for any system addressing its desiderata. The genesis for *AB-FFS Decoupled* is not fully specified in this MIP and such full specification is provided under [MIP-n](todo). However, generally, there are two strategies for genesis:
```

Comment:
> *AB-FFS Decoupled* is not fully specified

what is missing: either list it here, or point towards where does it say in this file what is missing.

Reply:
> [MD-38](https://github.com/movementlabsxyz/MIP/pull/38) requests provisions for genesis for any system addressing its desiderata.

requires to be more specific. what does this mean in this context in one sentence?

Reply:
> requires to be more specific. what does this mean in this context in one sentence?

[MD-38.D4](https://github.com/movementlabsxyz/MIP/blob/l-monninger/provisions-for-fixed-token-supply/MD/md-38/README.md#d4-provide-guides-and-software-for-genesis-procedures-and-initial-conditions-of-l1-and-l2-networks-using-the-atomic-bridge-and-fast-finality-settlement) requests that the any document proposing against its desiderata also specify a means for initializing a system.